### PR TITLE
docs: taxhub_url parameter is no longer configurable from UI

### PR DESCRIPTION
**Pull request by camillemonchicourt**
Also to remove in https://github.com/PnX-SI/gn_mobile_core/blob/develop/datasync/src/test/resources/fixtures/settings_sync.json ?
And other files from this repository ?

I also think it would be good to merge ``develop`` into ``master`` in this repository.

### CODE REVIEW

**Comment by sgrimault** [comment]
`docs/settings.adoc` should be deleted as well (no longer used) as it describes requested settings for old "Sync" app.
`master` branch will be updated from `develop` when an "official" release of "Occtax" app will be published.

File: datasync/README.md [hunk 1]
```diff
@@ -38,7 +38,6 @@ Module settings can be loaded from JSON as follow:
 {
   "sync": {
     "geonature_url": "https://demo.geonature/geonature",
-    "taxhub_url": "https://demo.geonature/taxhub",
```
Comment (sgrimault) [inline datasync/README.md]: `taxhub_url`parameter is always used as well as `geonature_url`.
These settings reflects what an app can receive from GeoNature server.

File: datasync/README.md [hunk 2]
```diff
@@ -53,7 +52,6 @@ or
 ```json
 {
   "geonature_url": "https://demo.geonature/geonature",
-  "taxhub_url": "https://demo.geonature/taxhub",
```
Comment (sgrimault) [inline datasync/README.md]: same here

File: datasync/README.md [hunk 3]
```diff
@@ -67,7 +65,6 @@ or
 | Parameter                         | UI      | Description                                          | Default value |
 | --------------------------------- | ------- | ---------------------------------------------------- | ------------- |
 | `geonature_url`                   | &#9745; | GeoNature URL                                        |               |
-| `taxhub_url`                      | &#9745; | TaxHub URL                                           |               |
```
Comment (sgrimault) [inline datasync/README.md:70]: Unlike the `geonature_url` parameter, `taxhub_url` is not configurable from UI.

**Review by sgrimault — CHANGES_REQUESTED** [review]

**Comment by camillemonchicourt** [comment]
OK I let you change that as I dont want to do mistakes.

**Pull request merged**